recipes/ignition-tools/all/conanfile.py

```diff
@@ -27,79 +29,104 @@ def _minimum_cpp_standard(self):
     def _minimum_compilers_version(self):
         return {
             "Visual Studio": "16",
+            "msvc": "192",
             "gcc": "7",
             "clang": "5",
             "apple-clang": "10",
         }
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
+    def requirements(self):
+        self.requires("backward-cpp/1.6")
```

Comment:
Why does `backward-cpp` is required only now? Its backward library in ignition-tools is not the same from backward-cpp project. https://github.com/gazebosim/gz-tools/tree/ignition-tools_1.4.0/src/Backward#installation